survey_app/build.gradle

```diff
@@ -114,10 +114,9 @@ dependencies {
     implementation 'com.android.support:support-annotations:27.0.2'
     implementation 'com.android.support:support-v13:27.0.2'
     implementation 'com.google.firebase:firebase-core:11.6.2'
-    implementation ('com.crashlytics.sdk.android:crashlytics:2.8.0@aar') {
-        transitive =true;
+    implementation('com.crashlytics.sdk.android:crashlytics:2.8.0@aar') {
+        transitive = true;
```

Comment:
Please remove whitespace changes like this and like lines 120, 131, etc. Please isolate the actual code changes.

Reply:
Sure, would be doing the required changes soon.